exasol/slc/internal/tasks/clean/clean_images.py

```diff
@@ -61,9 +63,9 @@ def is_child_image(self, possible_child, docker_client):
 
 
 class CleanImagesStartingWith(DockerBaseTask):
-    starts_with_pattern = luigi.Parameter()
+    starts_with_pattern: str = luigi.Parameter()  # type: ignore
```

Comment:
Maybe a utility function would be useful, e.g. `luigi_string_parameter() -> str` for wrapping `luigi.Parameter()`?
But maybe this reduces the readability of the code?

Reply:
Makes sense, but this kind of function should be implemented in the ITDE....